### DISCUSSION TITLE
I'm not seeing low energy 1/v for elastic scattering when I generate ACE files

### DISCUSSION THREAD

**Discussion by sombrereau** [njoy/NJOY21#174] [Q&A] [answered]
NJOY community,

I'm trying to create internally generated ACE files of updated nuclear data evaluations to use in MCNP. I'm able to create correctly formatted MCNP libraries. I was using Release of ENDF/B-VIII.0-Based ACE Data Files (LA-UR-18-24034 ) as a template for NJOY inputs. I used the inputs in Appendix A only changing the temperature from 293.6K to 300K and creating data for VIEWR / using VIEWR to create and post script image. When I look at my cross section libraries in VIEWR I see a subtle discrepancy in the behavior of elastic scattering in the low energy regions. Notice the green elastic scatter cross section line is flat at low energies.
![U235_principal_xs](https://user-images.githubusercontent.com/5852776/210003537-0e4ed285-1dbe-4af6-b7cf-b39d072ccfff.png)
I do not see the 1/v structure that I see in the ACE files distributed with MCNP6.2 or the cross sections I see on IAEA Nuclear Data Service Site: https://www-nds.iaea.org/exfor/endf.htm 
![U235_nel](https://user-images.githubusercontent.com/5852776/210003556-03cee056-be71-4246-8648-2e8a3c214064.png)

Am I missing a flag or setting? I'm using `u235ib46o28t6DNcnu5ef0STzt` version of U235 NDENS (https://www-nds.iaea.org/INDEN/).

**Comment by nathangibson14** [comment]
Hi @sombrereau ,

Unlike with capture reactions, the 1/v behavior from elastic scattering is an effect of Doppler broadening.  So my first thought is that you are either not plotting the correct temperature or something went wrong in your BROADR run.  Can you share the NJOY input you were using?  And can you also provide the ACE file you generated?  I could use some tools I have to compare with our typical inputs.  If that's not possible, I can whip together a more complete example at some point soon.

Thanks,
Nathan

**Comment by sombrereau** [comment]
Sure thing @nathangibson14 
I ran these three NJOY inputs in series: 

1. [u235.res.txt](https://github.com/njoy/NJOY21/files/10346431/u235.res.txt)
2. [u235.proc.txt](https://github.com/njoy/NJOY21/files/10346437/u235.proc.txt)
3. [u235.ace.txt](https://github.com/njoy/NJOY21/files/10346446/u235.ace.txt)

My ACE file is too large to upload here ... lemme figure something out.

**Reply by nathangibson14** [reply]
I just ran your inputs and I get the same plot.  I can work with this, no need to send the ACE file.  I'll let you know what I figure out.

**Comment by nathangibson14** [comment, marked as answer]
Hi @sombrereau,

Your third input file, `u235.ace.txt` uses tape numbers that assume some renaming from the first two input files.  This is something that is commonly done with automated tools at LANL, and so I'm sure this mistake stems from using our published template.  But if you look closely, you'll see the PENDF file input into PURR is `tape21`, which was created by `u235.res.txt` before the second input file ran.  Thus, you're running PURR with 0K cross sections.

Unfortunately, you also hit a not very user friendly problem here.  NJOY will actually check in ACER that you have the desired temperature on the PENDF that you pass into it.  However, PURR changed the temperature on the file to reflect the temperature the URR probability tables were sampled at.  So ACER thought that 300K was indeed present even though it wasn't.

The solution is to update the tape numbers in the last input file.  You have two appropriate PENDF files, which your second input names `tape22` and `tape23`.  Either would work, but you'd have to also update the rest of the tape numbers in the last input file.

The difference between those two is in one BROADR parameter: `thnmax`.  The negative value in the second execution tells BROADR to use the abs value of that parameter as the top energy to broaden to regardless of the presence of threshold reactions.  Our templates have both in there, as there are competing reasons to choose one over the other: there is a threshold below the top of the resonance region in recent U-235 evaluations, so broadening shouldn't stop there.  But broadening threshold reactions can mess up output spectra processing, so you also might not want to.  As you are doing your own processing, I'd recommend playing around with that option a bit to make sure you are happy with what you are getting for your application.

--Nathan

**Reply by sombrereau** [reply]
Thanks so much @nathangibson14. This is VERY helpful -- I think I can adjust the tape numbers around as you suggest and follow up with you.

**Reply by sombrereau** [reply]
@nathangibson14 -- I'm seeing the 1/v in the elastic scattering I was expecting. Thanks for your help.
![image](https://user-images.githubusercontent.com/5852776/210661015-d4995306-da50-4c30-8216-a9b9eea78aca.png)
